### PR TITLE
Fix startup issues and update documentation.

### LINKED ISSUE #20: nomad starts before network is ready
https://github.com/fredrikhgrelland/data-mesh/issues/59

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
closes #20

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -11,4 +11,8 @@ Vagrant.configure("2") do |config|
     vb.linked_clone = true
     vb.memory = 2048
   end
+  config.vm.provision "ansible_local" do |startup|
+    run = "always"
+    startup.playbook = "/etc/ansible/startup.yml"
+  end
```
Comment (m88614) [inline Vagrantfile:17]:
```suggestion
  config.vm.provision "ansible_local" do |startup|
    run = "always"
    startup.playbook = "/vagrant/ansible/playbook.yml"
  end
  config.vm.provision "ansible_local" do |startup|
    run = "always"
    startup.playbook = "/etc/ansible/startup.yml"
  end
```

**Comment by m88614** [comment]
> I spoke way too soon, sorry. I thought it provisioned properly, but it doesn't. It doesn't seem like the `playbook.yml` playbook has been run, because I get an error saying `startup.yml` doesn't exist.
> image

Man. This is obviously because I haven't built the new box (I assume). Building now and checking.

**Comment by fredrikhgrelland** [comment]
Should we merge this? Tests seems ok

**Comment by m88614** [comment]
Yeah. merge it! :D

**Comment by fredrikhgrelland** [comment]
Updated both ( vault and consul ) master tokens to `master` in latest push.

**Pull request merged**